labs/3-Add-Identity/README.md

```diff
@@ -121,7 +116,7 @@ Now that our Keycloak instance is setup as an IdP, we can configure the web site
     builder.Services.AddHttpClient(OpenIdConnectBackchannel, o => o.BaseAddress = new("http://idp"));
     ```
 
-1. In the same file, add the following methods that will configure authentication and authorization services in the application's DI container, and configure the OIDC authentication handler to use our IdP (remember to add any required `using` statements to import namespaces):
+1. In the same file, add the following methods that will configure authentication and authorization services in the application's DI container, and configure the OIDC authentication handler to use our IdP (remember to add any required `using` statements to import the `System.Security.Claims` namespace):
```

Comment:
Now you're just making it **too** easy for them :wink:

Reply:
Well, kinda, but since it's an extension method it doesn't pop up in the tooling.